src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
```

Comment:
Creating a square, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Instead of directly passing the `side` parameter to the constructor, consider declaring it beforehand and then pass the variable.